features/registerHealthCertificate.feature

```diff
@@ -0,0 +1,31 @@
+@RegistryService
+Feature: Register Health Certificate
+
+  @RegisterHealthCertificate
+  Scenario: Submit Valid Register Health Certificate Transaction
+    Given I make a POST request to /
+      And I set json body to the file at ./features/fixtures/registerHealthCertificate/Bundle-ProvideDDCCDocument.json
```

Comment:
This example should use the profile https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCProvideDocumentBundle.html, which is derived from the IHE.MHD.Minimal.ProvideBundle. The bundle entries must include a DDCCSubmissionSet and optionally may include DDCC Document References and a Patient as well. The slices are closed, so it cannot have List or Binary type entries.

Reply:
I updated `Bundle-ProvideDDCCDocument.json` in accordance with the DDCC Provide Document Bundle profile. For testing against the profile, I made a new step locally that validates the spec request against the profile. After some debugging I was able to get it to successfully validate against the profile.

Reply:
Some of the defined profiles in the sample are not correct - DDCC IG overwrites a lot of the IHE profiles. For example, http://profiles.ihe.net/ITI/MHD/StructureDefinition/IHE.MHD.Minimal.ProvideBundle should be http://worldhealthorganization.github.io/ddcc/StructureDefinition/DDCCProvideDocumentBundle.

Reply:
@nkarip Can you take a look at the profiles I changed within this file? I was able to change most of them over to the DDCC profiles, but for the [SubmissionSet profile](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCSubmissionSet.html) it looks like the profiles for the designation type and source id should be from IHE so I kept the same profile for those attributes.

Reply:
I pushed an updated example to the branch - please review the changes and update the other registerhealthcertificate scenario requests accordingly.